nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -138,13 +138,6 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - `services.avahi.nssmdns` got split into `services.avahi.nssmdns4` and `services.avahi.nssmdns6` which enable the mDNS NSS switch for IPv4 and IPv6 respectively.
   Since most mDNS responders only register IPv4 addresses, most users want to keep the IPv6 support disabled to avoid long timeouts.
 
-- `multi-user.target` no longer depends on `network-online.target`.
```

Comment:
This was only partially reverted, no?

Reply:
@RaitoBezarius the warning remains but the dependency is back. I figured we could put this back in the release notes when we remove the dependency again. But I can leave the bit about the warning in if you want.

Reply:
done